Sources/SwiftInspectorVisitors/Tests/ClassVisitorSpec.swift

```diff
@@ -246,6 +246,26 @@ final class ClassVisitorSpec: QuickSpec {
 
             expect(matching.count) == 1
           }
+
+          it("finds BarFooEnum") {
+            let matching = self.sut.innerEnums.filter {
+              $0.name == "BarFooEnum"
+                && $0.inheritsFromTypes == []
+                && $0.parentTypeName == "FooClass"
+            }
+
+            expect(matching.count) == 1
+          }
+
+          it("finds BarBarFooClass") {
```

Comment:
This class is the same name as another class nested under a struct. I wonder if we can make each type name in the code we're testing unique, for easier reading of these tests? Right now if you search a type short name you will find tests for different fully qualified types.

Reply:
Or we could have the name in the `it(...)` clause be fully qualified but that may get verbose.

Reply:
I made the `it` fully qualified in other tests. I'll do the same here.

265799912675f0555a89354a524afb27834bbe9e